File: codes/1524-string-matching-in-an-array/string-matching-in-an-array.py
class Solution(object):
    def stringMatching(self, words):
        """
        :type words: List[str]
        :rtype: List[str]
        """
        output=[]
        for i in words:
            for j in words:
                if i!=j and i in j:
                    output.append(i)
                    break
        return output

File: codes/1524-string-matching-in-an-array/string-matching-in-an-array.cpp
using namespace std;
class Solution {
public:
    vector<string> stringMatching(vector<string>& words) {
        vector <string> output={};
        for (string i: words){
            for (string j:words){
                if(i!=j and j.contains(i)){
                    output.push_back(i);
                    break;
                }
            
            }
        
        }
        return output;
    }
};